src/core/heating_cooling_loads.jl

```diff
@@ -47,8 +42,13 @@ function DomesticHotWaterLoad(;
     time_steps_per_hour::Int = 1,
     latitude::Real=0.0,
```

Comment:
@adfarth I don't think so because an advanced user (that is not using the Scenario constructor) could still inspect the code to see how to access the lat/lon inputs of the DomesticHotWaterLoad (same for SpaceHeatingLoad). We should leave `city` out for the same reason.

Reply:
Ok! Addressed in https://github.com/NREL/REopt.jl/pull/82/commits/ffc1394d83a574be4efcced133ac50879c99e0a9